app/main.py

```diff
@@ -2,0 +1,12 @@
+def copy_file(command: str) -> None:
+    if (not isinstance(command, str) or len(command) < 3
+            or command[:3] != "cp "):
+        return
+
+    cmd_list = command.strip().split(" ")
+    if len(cmd_list) < 3 or cmd_list[-2] == cmd_list[-1]:
+        return
+
+    with open(cmd_list[-2], "r") as file_in,\
+            open(cmd_list[-1], "w") as file_out:
+        file_out.write(file_in.read())
```

Comment:
This line reads the entire file into memory and then writes it to the destination file. This can lead to high memory usage if the file is large. Consider copying the file in chunks to reduce memory usage. This is also a performance issue.